### ISSUE TITLE
检测框可视化

### ISSUE THREAD

**Issue by xingxg613** [zhang-tao-whu/e2ec#46]
![1](https://github.com/zhang-tao-whu/e2ec/assets/175029193/d4d0cf96-7af7-4b2b-9fcb-d92c5856247c)
请问为什么检测框和分割的轮廓差别如此大？
检测框可视化的代码使用的是snake仓库中的代码，
box = output['detection'][:, :4].detach().cpu().numpy()
        for i in range(len(ex)):
            color = next(colors).tolist()
            poly = ex[i]
            poly = np.append(poly, [poly[0]], axis=0)
            ax.plot(poly[:, 0], poly[:, 1], color=color, linewidth=5)

            x_min, y_min, x_max, y_max = box[i]
            ax.plot([x_min, x_min, x_max, x_max, x_min], [y_min, y_max, y_max, y_min, y_min], color='w', linewidth=0.5)
希望可以收到回复，非常感谢
